tasks/instagrampoll.php

```diff
@@ -60,134 +60,137 @@ public function cleanup()
+							if($xml->Code == 'AccessDenied')						
+							{						
+								throw new ImageAccessDeniedException("Access denied to 'http://distilleryimage0.s3.amazonaws.com/47040426c57211e2880f22000a1f9ca7_5.jpg'. This entry hasn't been saved to the DB.");
+							}
 						
-						// Thrown an uknown error exception
-						throw new UknownInstagramError("Uknown error ({$xml->Code}): {$xml->Message}");
-					}												
+							// Thrown an uknown error exception
+							throw new UknownInstagramError("Uknown error ({$xml->Code}): {$xml->Message}");
+						}												
 					
-					// Carry on..
-					if(!$image) {
-						$image = \Propeller\Instagram\Model_Image::forge();
-						$image->thumb_img = $med->images->thumbnail->url;
-						$image->main_img = $med->images->standard_resolution->url;
-						$image->lowres_img = $med->images->low_resolution->url;
-						$image->instagram_id = $med->id;
-						$image->author = $med->user->username;
-						$image->link = $med->link;
-						$image->accepted = 'unsorted';
-						$image->subscription_id = $sub->instagram_subscription_id;
-						$image->caption = $med->caption->text;
-
-						$image->save();
-
-						//Loop Thorugh Tags and store each one
-						foreach($med->tags as $tag) 
+						// Carry on..
+						if(!$image) 
 						{
-							//Check Tag Existance
-							$check_tag_existance = \Propeller\Instagram\Model_Tag::query()->where('tag_name', $tag)->get_one();
-							if ($check_tag_existance) 
-							{
-								//Get Model Of Tag if it already exists
-								$tag_id = $check_tag_existance->id;
-								$tag_model = \Propeller\Instagram\Model_Tag::find($tag_id);
-							}
-							else
-							{
-								//Create That Tag if it doesnt already exist
-								$tag_model = \Propeller\Instagram\Model_Tag::forge();
-								$tag_model->tag_name = $tag;
-								$tag_model->save();
-							}
+							$image = \Propeller\Instagram\Model_Image::forge();
+							$image->thumb_img = $med->images->thumbnail->url;
+							$image->main_img = $med->images->standard_resolution->url;
+							$image->lowres_img = $med->images->low_resolution->url;
+							$image->instagram_id = $med->id;
+							$image->author = $med->user->username;
+							$image->link = $med->link;
+							$image->accepted = 'unsorted';
+							$image->subscription_id = $sub->instagram_subscription_id;
+							$image->caption = $med->caption->text;
 
-							//Save Tags 
-							$image->tags[] = $tag_model;
 							$image->save();
-						}
 
-						$count++;
-					}				
+							//Loop Thorugh Tags and store each one
+							foreach($med->tags as $tag) 
+							{
+								//Check Tag Existance
+								$check_tag_existance = \Propeller\Instagram\Model_Tag::query()->where('tag_name', $tag)->get_one();
+								if ($check_tag_existance) 
+								{
+									//Get Model Of Tag if it already exists
+									$tag_id = $check_tag_existance->id;
+									$tag_model = \Propeller\Instagram\Model_Tag::find($tag_id);
+								}
+								else
+								{
```

Comment:
Could we keep all braces for anything other than functions and classes on the same line please?